app/models/program.rb

```diff
@@ -41,6 +41,7 @@ def confirmed
 
   before_create :create_event_types
   before_create :create_difficulty_levels
+  before_save :check_languages_format
```

Comment:
The user get a generic error message. I am going to use validation to include more specific error messages.